components/navigation/MenuBlocks.js

```diff
@@ -10,15 +10,15 @@ export default function MenuBlocks ({
         items.map((item, index) => {
           const isExternalHref = item.href && item.href.startsWith('http');
           return (
-            <a 
-              href={item.comingSoon ? null : item.href} 
-              key={index} 
-              className="-m-3 p-3 flex items-start space-x-4 rounded-lg hover:bg-gray-50 transition ease-in-out duration-150" 
-              target={isExternalHref ? "_blank" : undefined} 
+            <a
+              href={item.comingSoon ? null : item.href}
+              key={index}
+              className="-m-3 p-3 flex items-start space-x-4 rounded-lg hover:bg-gray-50 transition ease-in-out duration-150"
+              target={isExternalHref ? "_blank" : undefined}
               rel={isExternalHref ? "noopener noreferrer" : undefined}
             >
               <div className={`flex-shrink-0 flex items-center justify-center h-10 w-10 rounded-lg border border-gray-800 bg-secondary-100 text-gray-900 sm:h-12 sm:w-12 ${item.comingSoon && 'opacity-50'}`}>
-                <item.icon className="h-6 w-6" />
+                <item.icon className="h-8 w-8" />
```

Comment:
Any specific reason why the size of the icons is increased?

Reply:
I think it's for more visibility. @mcturco suggested we do that in the comment section above. She'll have her reasons for that.

Reply:
Yep like @Mcnoble1 said, I increased the size of all the icons so that you can see the detail better. Eventually we will replace the icons in the other menus as well to match the new icons in this PR 😄